packages/react-native/src/index.tsx

```diff
@@ -28,7 +28,7 @@ const BdReactNative = BdReactNativeModule
     );
 
 export function init(key: string, options?: { url?: string }): void {
-  return BdReactNative.init(key, options?.url || 'https://api.bitdrift.io');
```

Comment:
Did you men to commit this change?

Reply:
Yeah I ended up moving the handling of the url down one layer since it made it easier to maintain an interface that worked with new and old gen

Reply:
OK I skimmed but all this stuff is very foreign to me so not clear where it went. 😂

Reply:
Ah OK I see it. I got tripped up that Kotlin doesn't have it. Must have the default somewhere else.